core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.spec.ts

```diff
@@ -385,6 +385,25 @@ describe('Topic editor tab directive', function() {
     expect(topicPracticeTabSpy).not.toHaveBeenCalled();
   });
 
+  fit('should not call the TopicUpdateService for practice tab if there are less than 10 questions '+
+   'in the topic, no matter what', function () {
+     // There are initially less than 10 questions in the topic
```

Comment:
For the best test, use this test that you've written to make sure that 9 practice questions are not allowed. 
```
$scope.skillQuestionCountDict = {"skill1": 3, "skill2": 6};
```